components/ActivityDetails/index.js

```diff
@@ -1,29 +1,41 @@
 import StyledImageComponent from "@/components/StyledImageComponent";
-import { TransparentBackLink } from "@/components/StyledLinks";
 import { Icon } from "@/components/Icon";
 import styled from "styled-components";
 import {
   StyledButton,
+  TransparentBackButton,
   TransparentFavoriteButton,
 } from "@/components/StyledButton";
 import { StyledEditLink } from "@/components/StyledLinks";
+import { useRouter } from "next/router";
 
 export default function ActivityDetails({
   activity,
   onDeleteActivity,
   isFavorite,
   onToggleFavorite,
 }) {
+  const router = useRouter();
+
   return (
     <StyledArticle>
       <ImageContainer>
-        <TransparentBackLink href="/" title="Back to all Activities">
+        <TransparentBackButton
```

Comment:
This button does not necessarily needs to function as a "go back" button. its main feature is that its transparent. So maybe you overspecified the naming here, in case you want to reuse this button in the future.

Reply:
Thanks. Because there is a "TransparentFavoriteButton" too, we would prefer to keep the naming for better allocation.